lnrpc/rpc.proto

```diff
@@ -2768,3 +2783,19 @@ message BakeMacaroonResponse {
     /// The hex encoded macaroon, serialized in binary format.
     string macaroon = 1 [json_name = "macaroon"];
 }
+
+message ChannelRevenueRequest{
+    ///  The outpoint (txid:index) of the funding transaction.
+    ChannelPoint channel_point = 1;
```

Comment:
I remember we wanted to move away from this `ChannelPoint` message in favor of the more generic `Outpoint`

Reply:
Worth adding a comment on the `ChannelPoint` message to use `Outpoint` instead? Even if we're not deprecating it, maybe just a note to use `Outpoint` where possible?

Reply:
Sounds good